### PR TITLE
Create CONTRIBUTING.md

### LINKED ISSUE #98: Create contribution guidelines

### PR DESCRIPTION
**Pull request by nathan-weinberg**
Fixes #98

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+Thank you for your interest in contributing to Jeeves!
+
+## Getting Started
+Every contributor must do the following before getting started:
+
+1. **Become a Jeeves Collaborator:** Only GitHub Collaborators in the Jeeves repo can submit code, so if you aren't already, reach out to [Nathan Weinberg](mailto:nweinber@redhat.com) to recieve an invite giving you the appropriate permissions.
+
+2. **Fork the Jeeves repo:** All work being done **must** be done within your own fork on a working branch (i.e. not `master`). To submit your work, make a Pull Request from your fork's working branch to the `master` branch of the main repo (more on this below). **Do not** push branches directly to the main repo for the project; they will be deleted without consultation.
+
+3. **Set your remotes properly:** Make sure your remotes are set such that `git remote -v` gives you an output similar to the following:
```
Comment (EliadCohen) [inline CONTRIBUTING.md:12]:
```suggestion
3. **Set your remotes properly:** After cloning your forked repo (`git clone git@github.com:<your-github-username>/jeeves.git`), make sure your remotes are set such that `git remote -v` gives you an output similar to the following:
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+[jeeves]$ git checkout master				# checkout your master branch
+[jeeves]$ git pull upstream master			# update your local master branch with latest changes from upstream
+[jeeves]$ git push origin master			# update your remote master branch with latest changes
+[jeeves]$ git checkout -b <working-branch>		# create your new working branch containing latest changes
+```
+
+## Submitting your change via making a Pull Request
+Before you make a Pull Request, make sure you do the following:
+
+1. **Test your code:** Make sure you test your code **thouroughly** before submitting; Pull Requests containing broken code will **not** be looked upon favorably.
```
Comment (EliadCohen) [inline CONTRIBUTING.md:42]: Maybe soften it up a bit :)

File: CONTRIBUTING.md (hunk 1 above)
Comment (nathan-weinberg) [reply CONTRIBUTING.md:12]: Will change in next patchset!

File: CONTRIBUTING.md (hunk 2 above)
Comment (nathan-weinberg) [reply CONTRIBUTING.md:42]: Less **bold** you might say? ;)

Comment (EliadCohen) [reply CONTRIBUTING.md]: That's a start. Maybe say: We want you to take pride in good and effective code that requires minimal to zero changes during your pull request's review.

Or "Contributing good code not only makes your pull requests pass faster but also helps oyur community credibility"

**Pull request merged**